plotnine/geoms/geom_text.py

```diff
@@ -177,7 +180,9 @@ def draw_group(data, panel_params, coord, ax, **params):
                 kw['bbox'] = bbox
                 kw['bbox']['edgecolor'] = params['boxcolor'] or kw['color']
                 kw['bbox']['facecolor'] = kw.pop('facecolor')
-            ax.text(**kw)
+            text_elem = ax.text(**kw)
+            if params['path_effects']:
+              text_elem.set_path_effects(params['path_effects'])
```

Comment:
You need to add 'path_effects' to the `DEFAULT_PARAMS` above.

Reply:
Done 👍